events-api/src/shared/validators/cardValidators/card-holder-name.validator.ts

```diff
@@ -0,0 +1,14 @@
+import { ValidatorConstraint, ValidatorConstraintInterface } from 'class-validator';
+
+@ValidatorConstraint({ name: 'cardHolderName', async: false })
+export class CardHolderNameConstraint implements ValidatorConstraintInterface {
```

Comment:
Better to use a function rather than a class. It is more readable and more common approach